Date: Mon, 4 May 2009 16:17:46 -0400
From: "Matthew L. Creech" <mlcreech@gmail.com>
To: boost@lists.boost.org
Subject: [boost] [PATCH] Typo in bootstrap.sh

Boost fails to install when LIBDIR isn't specified explicitly and
isn't "/lib", due to a typo in bootstrap.sh.  Trivial patch below.


--- boost_1_39_0.orig/bootstrap.sh
+++ boost_1_39_0/bootstrap.sh
@@ -249,7 +249,7 @@
 fi
 
 if test "x$LIBDIR" = x; then
-  LIBDIR="$EXPREFIX/lib"
+  LIBDIR="$EPREFIX/lib"
 fi
 
 if test "x$INCLUDEDIR" = x; then
